test/Scrutor.Tests/ScanningTests.cs

```diff
@@ -140,7 +175,7 @@ public void AutoRegisterAsMatchingInterface()
                     .AsMatchingInterface()
                     .WithTransientLifetime());
 
-            Assert.Equal(2, Collection.Count);
+            Assert.Equal(3, Collection.Count);

```

Comment:
This is OK. `AsMatchingInterface` was a community contribution that allows you to automatically register `DuplicateInheritance` as `IDuplicateInheritance` by name. In this case, this is 100% correct :+1: